draft-ietf-webpush-protocol.xml

```diff
@@ -951,7 +953,7 @@ Host: push.example.net
           Each receipt is pushed as the response to a synthesized GET request sent in a
           PUSH_PROMISE.  This GET request is made to the same push message resource that
           was created by the push service when the application server requested message
-          delivery. A successful response includes a 204 (No Content) status code with no data.
+          delivery. The response includes a status code indicating the result of the message delivery and carries no data.

```

Comment:
I think nothing is removed as the 204 requirement is in the next paragraph (after the example).
I removed it from here to make this paragraph more generic: what is in the pushed response, leaving the next two paragraphs dealing with successful and unsuccessful delivery.